boxoffice/extapi/slack.py

```diff
@@ -0,0 +1,28 @@
+from boxoffice import db, app
+from boxoffice.models import ItemCollection, LineItem, Order, ORDER_STATUS
+from datetime import datetime as dt
+import requests
+from urllib import urlencode
+import json
+from tabulate import tabulate
+from rq.decorators import job
+
+@job('boxoffice')
+def post_stats(order_type, id, webhook_url):
+    if id is None:
+        return None
+
+    with app.test_request_context():
+        item_collection =  ItemCollection.query.get(id)
+        started = [ORDER_STATUS.PURCHASE_ORDER]
+        sold = [ORDER_STATUS.SALES_ORDER, ORDER_STATUS.INVOICE]
+        cancelled = [ORDER_STATUS.CANCELLED]
+        results = []
+        for item in item_collection.items:
+            started = LineItem.query.join(Order).filter(LineItem.item == item, Order.status.in_(started)).count()
```

Comment:
You've already used `started` as a variable on line 17. `started_line_items` would be better. Same with Lines 23 and 24.